pkg/applicationmanager/controllers/applicationrestore.go

```diff
@@ -483,88 +512,167 @@ func (a *ApplicationRestoreController) restoreVolumes(restore *storkapi.Applicat
 	namespacedName.Namespace = restore.Namespace
 	namespacedName.Name = restore.Name
 	restoreCompleteList := make([]*storkapi.ApplicationRestoreVolumeInfo, 0)
+	nfs, err := IsNFSBackuplocationType(backup)
+	if err != nil {
+		logrus.Errorf("error in checking backuplocation type")
```

Comment:
It is better to error out and fail the backup. Other wise, we might be taking non nfs, even if it is nfs.